### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Khanetmoi**
### In this project I followed the following instructions:

- You will find all the details of the Bookstore design in [Zeplin](https://zeplin.io/).
    [Login to Zeplin](https://app.zeplin.io/login) (hint: you will find the credentials in the [sneak peek](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/sneak_peek_v2_0.md) of the project).  
 - Open the project Bookstore CMS.
 -   Add any missing JSX elements to your projects (without adding extra functionality)
 -   You may use the CSS rules detailed in Zeplin to style the website.
 -  Your final design should match Zeplin's design (this includes all elements + the same styling)

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -25,3 +25,46 @@ code {
...
+  box-shadow: 1px 1px 1px rgb(218, 214, 214);
+  align-items: center;
+}
+
+.logo {
+  text-decoration: none;
+  font-size: 2rem;
+  font-weight: 600;
+  color: rgb(51, 182, 226);
+}
```
Comment (levy002) [inline src/index.css]: - [ ] Kindly make sure the logo color is the same as the one on the provided Figma design.

   - ### YOUR DESIGN
   
      image

   - ### FIGMA DESIGN
   
      image

File: src/components/books/books.css [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  display: flex;
+  gap: 1rem;
+}
+
+.callContainer > button {
+  padding: 0.6rem;
+  border: none;
+  color: blue;
+}
+
```
Comment (levy002) [inline src/components/books/books.css]: - [ ] Kindly make sure your button styles match the provided Figma design. Typography, color, and background color kindly make sure to match the design.
- [ ] Please add the vertical line which is between every two buttons in order to meet the provided Figma design.

   - ### YOUR DESIGN
   
     image

   - ### FIGMA DESIGN
   
      image

File: src/components/books/books.css [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+.bookContent > h2 {
+  margin: 2px;
+}
+
+.bookContent p {
+  margin-top: 2px;
+  color: blue;
+}
```
Comment (levy002) [inline src/components/books/books.css]: - [ ] Kindly make sure to use the same color use on the provided design.

    - ### YOUR DESIGN
     image

    - ### PROVIDED DESIGN
    
     image

File: src/components/books/books.css [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+form {
+  display: grid;
+  grid-template-columns: 1fr 0.5fr 0.2fr;
+  gap: 1rem;
+  width: 80%;
+}
+
+input {
+  padding: 0.8rem;
+}
```
Comment (levy002) [inline src/components/books/books.css:132]: - [ ] Kindly make sure your input field looks as much as possible to the provided design.

   - ### YOUR DESIGN
   
       image

   - ### PROVIDED DESIGN
   
     image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @Khanetmoi   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Professional PR description :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] Kindly make sure your buttons' styles match the provided design as much as possible.

   - ### YOUR DESIGN
   
    image
    image

   - ### PROVIDED DESIGN
      image
      image

- [ ] Kindy make sure your main content of the website is aligned on the same line vertically in order to meet the provided design.

  - ### YOUR DESIGN
  
    image

  - ### PROVIDED DESIGN
  
     image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
## Hi @Khanetmoi,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- Professional readme file ✔️
- Amazing design ✔️
- No linter errors ✔️
209362895-9762f2f8-13cf-4ce0-90b6-a672c4f04196

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**